### PR TITLE
[TE-16.3] Test specification for encap FRR with REPAIR

### PR DESCRIPTION
**Pull request by nachikethas**
Adds the test specification for FRR traffic flows using the indirection of the `REPAIR` VRF instead of specifying the backup action directly in the next hop.

### CODE REVIEW

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+BGP/IS-IS routes in the DEFAULT VRF.
+
+1.  Update `NHG#1000` to the following:
+
+```
+NHG#1000 (Default VRF) {
+        {NH#1000, DEFAULT VRF}
+}
+NH#1000 -> {
+  decapsulate_header: OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4
```
[the remark this answers is not in the document]
Comment (nachikethas) [reply feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:423]: Please see Test-8 "no match in TE_VRF_222" for the test for that specific flow.

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+2.  Shutdown DUT port-2, port-3, and port-4, and port-6.
+3.  Validate that all traffic is no longer encapsulated, and is all egressing
+    out of DUT port-8 per the BGP-ISIS routes in the default VRF.
+
+#### Test-7, no match in encap VRF
+
+Test that if there is no lookup match in the encap VRF, then the traffic should
+be routed to the DEFAULT VRF for further lookup.
+
+1.  In `ENCAP_TE_VRF_A`, Add an 0/0 static route pointing to the DEFAULT VRF.
```
Comment (nflath) [inline feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:495]: This is going to be done via config rightr?

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+2.  Send traffic with destination address 20.0.0.1, which should produce no
```
Comment (nflath) [inline feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:496]: Do we need to specify QoS?

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nflath) [reply feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:423]: Ah yes, this is tested later.  I don't think we're currently planning on using this, are we just leaving it here so when we can simplify?

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md (hunk 2 above)
Comment (nachikethas) [reply feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:495]: That's correct. I can add "... using gNMI" to be more prescriptive if you like.

Comment (nachikethas) [reply feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md]: done

File: feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md (hunk 3 above)
Comment (nachikethas) [reply feature/gribi/otg_tests/encap_frr_with_repair_vrf/README.md:500]: done

**Comment by nflath** [comment]
LGTM

**Review by nflath — APPROVED** [review]

**Review by xw-g — APPROVED** [review]

**Pull request merged**